lib/services/task_list_service.dart

```diff
@@ -26,6 +27,58 @@ class TaskListService with LogMixin, ChangeCallbackProvider {
     return (await _taskListCrdt).values;
   }
 
+  Future<List<Task>> getTasksForList(TaskList taskList) async {
+    var filtered = <Task>[];
+
+    List allTasks = await tasks;
+    for (var i = 0; i < allTasks.length; i++) {
+      if (allTasks[i].taskListID == taskList.id) {
+        filtered.add(allTasks[i]);
+      }
+    }
+
+    switch (taskList.sortBy) {
+      case SortOption.Title:
+        filtered
+            .sort((a, b) => a.title.toString().compareTo(b.title.toString()));
+        break;
+      case SortOption.Flag:
+        filtered.sort((a, b) {
+          if (b.isFlagged) {
+            return 1;
+          }
+
+          return -1;
+        });
+        break;
+      case SortOption.Status:
+        filtered.sort((a, b) {
```

Comment:
`return b.completed ? -1 : 1`
(Btw. does it have to be -1 or would it also work with `b.completed ? 0 : 1`?)

Reply:
good point :D works with 0